WS.js

```diff
@@ -31,7 +34,8 @@ function WS(uri,protocols){
         wsList[uri+protocols]=newWS;
     }
     var ws=wsList[uri+protocols];
-
+    ws.addEventListener = ws.addListener;
+    ws.removeEventListener = ws.removeListener;
```

Comment:
similar to above

This seems like it will overwrite the browser functionality when run in the browser, did you test this?

If it does, maybe there should be a seperate if(isNode)